core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-scheduling-add/dot-experiments-configuration-scheduling-add.component.ts

```diff
@@ -0,0 +1,100 @@
+import { Observable } from 'rxjs';
+
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms';
+
+import { ButtonModule } from 'primeng/button';
+import { CalendarModule } from 'primeng/calendar';
+import { CardModule } from 'primeng/card';
+import { SelectButtonModule } from 'primeng/selectbutton';
+import { SidebarModule } from 'primeng/sidebar';
+
+import { take } from 'rxjs/operators';
+
+import { DotFieldValidationMessageModule } from '@components/_common/dot-field-validation-message/dot-file-validation-message.module';
+import { RangeOfDateAndTime, Status, StepStatus } from '@dotcms/dotcms-models';
+import { DotMessagePipeModule } from '@pipes/dot-message/dot-message-pipe.module';
+import { DotExperimentsConfigurationStore } from '@portlets/dot-experiments/dot-experiments-configuration/store/dot-experiments-configuration-store';
+import { DotSidebarDirective } from '@portlets/shared/directives/dot-sidebar.directive';
+import { DotSidebarHeaderComponent } from '@shared/dot-sidebar-header/dot-sidebar-header.component';
+
+@Component({
+    selector: 'dot-dot-experiments-configuration-scheduling-add',
+    standalone: true,
+    imports: [
+        CommonModule,
+        ReactiveFormsModule,
+
+        DotFieldValidationMessageModule,
+        DotMessagePipeModule,
+        DotSidebarHeaderComponent,
+        DotSidebarDirective,
+        //PrimeNg
+        SidebarModule,
+        ButtonModule,
+        SelectButtonModule,
+        CardModule,
+        CalendarModule
+    ],
+    templateUrl: './dot-experiments-configuration-scheduling-add.component.html',
+    styleUrls: ['./dot-experiments-configuration-scheduling-add.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DotExperimentsConfigurationSchedulingAddComponent implements OnInit {
+    form: FormGroup;
+    scheduling: RangeOfDateAndTime;
+    statusList = Status;
```

Comment:
You using this?

Reply:
yes in the template, rename variable to `stepStatus`